### PR TITLE
DOC-1993: Added TINY-9669 to the release notes.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-1993

Changes:
* Added TINY-9669 to the release notes.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] ~`modules/ROOT/nav.adoc` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by spocke — CHANGES_REQUESTED** [review]
Not sure how this ended up in the docs. The TinyState is a internal test module and should not be documented externally.

**Comment by kemister85** [comment]
> Not sure how this ended up in the docs. The TinyState is a internal test module and should not be documented externally.

The changelog entry for this ticket was as below.

> ### Added
- Added `TinyState` module that contains transactional functions for editor states.

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 1]
```diff
@@ -59,6 +59,14 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
...
 {productname} 6.5 also includes the following additions:
 
+=== Added `TinyState` module that contains transactional functions for editor states.
+//#TINY-9669
+
+In previous versions of {productname}, `TinyState` module was introduced, which includes transactional functions designed for editor states. This addition addressed an issue where toolbar items were not being disabled correctly when selecting `contenteditable= "false"` contents in a `contenteditable= "false"` root.
+
+As a consequence, toolbar buttons and menu items remained enabled even when they couldn't be applied to the current selection.
+
+In {productname} 6.5, a fix was implemented that ensures that toolbar buttons and menu items are now properly disabled based on the selection. As a result, when `contenteditable= "false"` content is selected in a `contenteditable= "false"` root, the toolbar and menu items are disabled. In addition, if `contenteditable= "true"` contents are selected, the toolbars and menu items are re-enabled.
```
Comment (kemister85) [inline modules/ROOT/pages/6.5-release-notes.adoc:69]:
```suggestion
=== Toolbar and menu items were not being disabled correctly when selecting noneditable elements inside a `contenteditable= "false"` root.
//#TINY-9669

In previous versions of {productname}, an issue was identified where toolbar and menu items were not being disabled correctly when selecting `contenteditable= "false"` contents inside a `contenteditable= "false"` root.

As a consequence, toolbar buttons and menu items would appear to be enabled even though the buttons and menu items were in a disabled state based on the users current selection.

A fix was applied in {productname} 6.5, that now disables the toolbar buttons and menu items based on the users selection. In addition, if `contenteditable= "true"` contents are selected, the toolbars and menu items are re-enabled.
```

Comment (kemister85) [reply modules/ROOT/pages/6.5-release-notes.adoc:69]: @spocke Please see above re-write suggestion.

**Review by spocke — APPROVED** [review]

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 2]
```diff
@@ -59,6 +59,14 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
...
 {productname} 6.5 also includes the following additions:
 
+=== Toolbar and menu items were not being disabled correctly when selecting noneditable elements inside a `contenteditable= "false"` root.
+//#TINY-9669
+
+In previous versions of {productname}, an issue was identified where toolbar and menu items were not being disabled correctly when selecting `contenteditable= "false"` contents inside a `contenteditable= "false"` root.
+
+As a consequence, toolbar buttons and menu items would appear to be enabled even though the buttons and menu items were in a disabled state based on the users current selection.
+
+A fix was applied in {productname} 6.5, that now disables the toolbar buttons and menu items based on the users selection. In addition, if `contenteditable= "true"` contents are selected, the toolbars and menu items are re-enabled.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.5-release-notes.adoc:69]:
```suggestion
=== Toolbar and menu items were not disabled correctly when selecting elements with a `contenteditable="false"` attribute inside an editor root with a `contenteditable="false"` attribute set.
//#TINY-9669

In previous versions of {productname}, an issue was identified where toolbar and menu items were not presenting as disabled when the contents of elements with a `contenteditable= "false"` attribute were selected from inside a editor root with a `contenteditable="false"` attribute set.

In this circumstance, toolbar buttons and menu items appeared as enabled even though the buttons and menu items were in a disabled state based on the user’s current selection.

{productname} 6.5 now explicitly disables toolbar buttons and menu items based on the user’s selection.

In addition, if the contents of elements with a `contenteditable="true"` attribute are selected in the same circumstance, the toolbars and menu items are re-enabled.
```

1. Edits and copy-edits.
2. The technically accurate phrases edited into place: _`contenteditable="false"` attribute_ and _`contenteditable="true"` attribute_

**Review by brian-forte — APPROVED** [review]

**Pull request merged**